### PR TITLE
Implement toolbar & menubar (initial ver.)

### LINKED ISSUE #4: Implement toolbar component to control dotting
We need controls to modify pixels when using dotting. For example, changing colors, changing brush modes.
It is necessary to create a `toolbar` component that can control dotting.

### PR DESCRIPTION
**Pull request by lerrybe**
### Related Issue
closed #4

### Changed
* Implement dotting context (current data: `isGridFixed`, `isGridVisible`, `isPanZoomable`)
* Attach controls to enable dotting editing (Implement UI of toolbar, menubar with tailwind).

### Next up (todo)
* Synchronization `clear`, `undo`, `redo` information with the yorkie remote.
* Synchronization related to indices.
* Allowing to look traces of each Client.

### Screenshot

스크린샷 2023-08-20 오후 2 23 27

### CODE REVIEW

File: src/assets/logo.svg
Comment (hunkim98) [inline src/assets/logo.svg:1]: This is crazy!💪

File: src/components/ControlTools.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import { BiRedo, BiUndo } from 'react-icons/bi';
+import { MdOutlineAddPhotoAlternate } from 'react-icons/md';
+import { TOOL_GIRD_CLASS, TOOL_TEMPLATE_CLASS } from '../styles/styleClass.js';
+
+const ControlTools = forwardRef(function InnerToolbar({}, ref: ForwardedRef<DottingRef>) {
+  // @ts-ignore
+  const { undo, redo, clear } = useDotting(ref);
+
+  useEffect(() => {
+    const keyboardUndoListener = (e: KeyboardEvent) => {
```
Comment (hunkim98) [inline src/components/ControlTools.tsx]: From what I know, `dotting` already has an undo and redo listener!

https://github.com/hunkim98/dotting/blob/761ac38987ab1651ae7e6cc32101f0b4d43f668a/src/components/Dotting.tsx#L867-L872

If you go to the link above. You can see that there is a keydown and keyup listener in the parent div. Remind that the key down and key up event is activated when you have clicked `Dotting` canvas

**Review by hunkim98 — APPROVED** [review]
Your code is very well arranged, and I am very impressed! Great work!!💪

Seems like you are thinking of allowing users to see other people's stroke in real-time (pixel by pixel). This necessitates the use of `presence` of yorkie. Unfortunately, there is no listener for the strokes that occur in the interaction canvas of `Dotting`. There is only a strokeEnd listener that is called when the data drawn in the interaction canvas is relayed to the data canvas. 

If you want to allow presence updates to be shown on each canvas, you could create an issue in the `dotting` repository and create a `strokingPixelListener` (or whatever you name it). It will not be that hard to implement it because there are many codes you can refer to create an event listener. If you are interested in the issue, you could do it on your own, or I could help you if you need it urgently!

And again, thank you for the great work. I really appreciate it🙏

**Comment by lerrybe** [comment]
@hunkim98 
Thank you so much for your thoughtful reply! 
I'll remove the unnecessary parts and complete this work that I haven't implemented yet to complete collab-dotting example ver.1. 🐱

**Pull request merged**